src/core/scene/a-scene.js

```diff
@@ -449,6 +457,25 @@ module.exports = registerElement('a-scene', {
         var effect = this.effect;
         var timeDelta = time - this.time;
 
+        // Experimental versions of Chromium used for WebVR have a known issue
```

Comment:
aargh must have started from a branch that had this, will back it out (second one today)